docs/user-guide/systemrequirements.md

```diff
@@ -123,10 +127,10 @@ The Zowe Desktop supports Google Chrome, Mozilla Firefox, Apple Safari and Micro
 For Firefox, both the regular and Extended Support Release (ESR) versions are supported under this rule.
 
 Currently, the following browsers are supported:
-    -   Google Chrome V77 or later
+    -   Google Chrome V77, V79 or later
     -   Mozilla Firefox V68 or later
-    -   Safari V12.1 or later
-    -   Microsoft Edge 18
+    -   Safari V12.1, V13 or later
```

Comment:
We need to remove the old versions, not append new ones. It's just following the explanation paragraph above.
Alternatively, remove all of the versions and just let the above paragraph explain which versions are supported.

Reply:
Fixed. The list is kept here so users don't have to spend time checking which release is the newest version.